pom.xml

```diff
@@ -16,7 +16,7 @@
         <!-- check micrometer.version vertx-micrometer-metrics consumes before bumping up -->
         <micrometer.version>1.1.0</micrometer.version>
         <junit-jupiter.version>5.7.0</junit-jupiter.version>
-        <uid2-shared.version>1.2.1</uid2-shared.version>
+        <uid2-shared.version>1.11.0</uid2-shared.version>
```

Comment:
Don't forget to update this version once you build out the new uid2-shared release